govuk-docker.gemspec

```diff
@@ -0,0 +1,30 @@
+# -*- encoding: utf-8 -*-
+
+lib = File.expand_path("lib", __dir__)
+$:.unshift lib unless $:.include?(lib)
+
+require "govuk_docker/version"
+
+Gem::Specification.new do |s|
+  s.name         = "govuk-docker"
+  s.version      = GovukDocker::VERSION
+  s.platform     = Gem::Platform::RUBY
+  s.authors      = ["GOV.UK Dev"]
+  s.email        = ["govuk-dev@digital.cabinet-office.gov.uk"]
+  s.summary      = "Docker-based GOV.UK environment"
+  s.homepage     = "http://github.com/alphagov/govuk-docker"
+  s.description  = "A local environment for GOV.UK powered by Docker"
+
+  s.required_ruby_version = ">= 2.6.0"
+  s.files        = Dir.glob("exe/**/*") + Dir.glob("lib/**/*") + %w(README.md)
+  s.bindir       = "exe"
+  s.executables  = s.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  s.require_path = "lib"
+  s.add_dependency "colorize"
```

Comment:
`gem build` recommends using a bounded requirement here. Such as `~> 0.8`

Reply:
I chose to leave them out because it currently matches the `Gemfile` so we get the same behaviour before and after. I was thinking this could be fixed in a subsequent PR, but perhaps I can just go ahead and fix it now in this PR?

Reply:
A subsequent PR works for me.